### DISCUSSION TITLE
How is penpot.app deployed at prod?

### DISCUSSION THREAD

**Discussion by ghost** [uxbox/uxbox#2085] [Q&A]
- Is it on docker-compose?
- How to handle zero down time update when release a new version?

More details would be appreciated.

**Comment by niwinz** [comment]
I think this question is better to ask in the new https://community.penpot.app/
I'm happily answer it there.

**Comment by ghost** [comment]
Thanks. Aksed there.

**Reply by jemedeiros** [reply]
Link to discussion: https://community.penpot.app/t/how-is-penpot-app-deployed-at-prod/312/3
